README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+My name is Mackenzie Whitfield.  
+I moved to Chicago from San Diego two weeks ago. I **love** the beach. My hobbies also include hiking, traveling, reading, and watching movies. Up until now, I have only lived in California.  
+
+# Where I have Lived 
+1. San Diego  
+2. San Francisco  
+3. Los Angeles
+4. Chicago
+
+# Education
+I studied psychology at Loyola Marymount University in Los Angeles. I began participating in undergraduate research my freshman year and completed an Honors Thesis in my senior year. One of the *most* memorable experiences I had as an undergraduate was my study abroad in Greece. Below is a link to the study abroad page specific to my program. If you go under Student Testimonials, you will see the one I wrote! 
+[LMU Odyssey Program](https://bellarmine.lmu.edu/moderngreek/beyondtheclassroom/studyabroad)
+
+# Future Plans  
+I am interested in a career in applied behavioral research, specifically in consumer psychology. While I do not know exactly where I might like to work, I look forward to spending my MAPSS year exploring my options, acquiring relevant skills, and networking with faculty and students.  
+
```

Comment:
I really liked your work and enjoyed how you sectioned off each topic, maybe using differently sized headers could make it more aesthetically pleasing?